src/server.js

```diff
@@ -0,0 +1,8 @@
+const app = require('./app.js');
+// const { seq } = require('../database/models/index.js');
+
+// seq.then(() => {
+app.listen(app.get('port'), () => {
+                console.log('Our app running  on port', app.get('port')); //eslint-disable-line
+});
+// }).catch(err => console.log('err', err));     //eslint-disable-line
```

Comment:
There is no need for the comment out code here beside that good job :+1: .

Reply:
Done :+1: